annotations/annotation-user/pom.xml

```diff
@@ -27,6 +27,13 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        
+        <dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.16.10</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
Sure, but the idea here is not to modify the underlying logic - just to fix the issues that pop up. 
For example, this code is used here: http://www.baeldung.com/java-annotation-processing-builder
Actually, this is an important point across the board - we have to be careful modifying the code because it is likely used by an article on the site. 
Hope that helps. Cheers, 
Eugen.